.github/workflows/test-update.yml

```
@@ -0,0 +1,327 @@
+        run: |
+          LATEST=$(curl -s https://api.github.com/repos/${{ github.repository }}/releases/latest | grep '"tag_name"' | cut -d'"' -f4 | sed 's/^v//')
+          PREVIOUS=$(curl -s https://api.github.com/repos/${{ github.repository }}/releases | grep '"tag_name"' | cut -d'"' -f4 | sed 's/^v//' | sed -n '2p')
+          echo "PREV_VERSION=$PREVIOUS" >> $GITHUB_OUTPUT
+          echo "NEW_VERSION=$LATEST" >> $GITHUB_OUTPUT
+
+      - name: Display test versions
+        run: |
+          echo "Testing cargo install update from v${{ steps.versions.outputs.PREV_VERSION }} to v${{ steps.versions.outputs.NEW_VERSION }}"
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.9)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)